### PR TITLE
Reserve a Java 9 module name

### PR DESCRIPTION
**Pull request by ljacomet**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -201,6 +201,17 @@
           <excludePackageNames>org.terracotta.statistics.jsr166e</excludePackageNames>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <archive>
+            <manifestEntries>
+              <Automatic-Module-Name>org.terracotta.statistics</Automatic-Module-Name>
```
[the remark this answers is not in the document]
Comment (ljacomet) [reply pom.xml]: It is not recommended, but it is not forbidden either.

**Comment by ljacomet** [comment]
I have rebased, up to you @chrisdennis based on the comments from @mathieucarbou

**Comment by chrisdennis** [comment]
To my mind the primary purpose of this library is 'statistics' and the fact that the other package is here is unfortunate but is a fait accompli.

**Comment by mathieucarbou** [comment]
@chrisdennis : why don't we move the `context` package inside statistics ? 
`context` is internal I think, and this is only used to store observers in Ehcache.

**Pull request merged**